js/classes/entities/foreground/hud.js

```diff
@@ -22,6 +23,37 @@ class Hud {
         return 10;
     }
 
+    swapToCrosshair() {
+        let crosshairCursorUnclicked = 'url(../sprites/crosshair_unpressed.png) 16 16, auto';
+        let crosshairCursorClicked = 'url(../sprites/crosshair_pressed.png) 16 16, auto';
+
+        document.body.style.cursor = crosshairCursorUnclicked;
+
+        document.body.addEventListener('mousedown', () => {
+            document.body.style.cursor = crosshairCursorClicked;
+        });
+
+        document.body.addEventListener('mouseup', () => {
+            document.body.style.cursor = crosshairCursorUnclicked;
+        });
+    }
+
+    swapToPointer() {
+        let pointerCursorUnclicked = 'url(../sprites/pointer_unpressed.png), auto';
+        let pointerCursorClicked = 'url(../sprites/pointer_pressed.png), auto';
```

Comment:
Same thought about naming PNGs consistent with the associated variables that I mentioned on line 28.